doc/yeast-capture.md

```diff
@@ -0,0 +1,77 @@
+# Yeast capture statistical estimations
+
+This is not a standard procedure; just general thoughts on priors estimation for a first slant.
+
+## Starting media
+
+Estimating yeast correspondence to particular expected strain on capture makes little sense; this would require us to trust manufacturer, to make assumptions on bottling process quality, some beers may have lagering yeast that is different from strain expected in style.
+
+Laboratory yeast, of course, states some strain identification, but first, we capture commercial pure cultures from living beer fermentation, and then, whatever is listed by other laboratories is based on their less then open standards of processing and is thus below data quality we aspire to achieve here.
+
+What we focus on here is yeast purity after capture, no matter how captured strain relates to origin.
+
+## Initial capture
+
+The yeast in starting media is often in poor condition and might be stuck to the walls. To pick it into solution, we've found the following to be effective:
+
+1. Prepare approx. 50-200 ml of sterile approx. 1.040 pale wort
+2. Empty the container almost to the bottom, so that 10-50ml remains
+3. Flame the opening and add sterile wort into container; mix thoroughly
+4. Wait until fermentation is visible; yeast could be sampled at high krauzen.
+5. To sample, we again flame the opening of the bottle and pour some wort into sterile test tube. This wort could be used to streak the plate.
+
+## Plate streak
+
+The colonies on plate are observed as circles, often isolated, indicating effective single-cell distribution.
+
+TODO: analyze plate photographs to obtain some information of colonies distribution
+
+TODO: estimate how well we could identify overlapping circles
+
+TODO: literature reference on this
+
+PRIOR
+Worst case estimate encountered in [Yeast: White, Zainacheff](ISBN-13: 978-0937381960) seems to be around 20% probability of pickup of colony from more than one origin cell. We will use worst-case for now.
```

Comment:
`Worst case estimate encountered in [^White-Zainacheff] seems to be around 20% probability of pickup of colony from more than one origin cell. `